### PR TITLE
Hide tabline

### PR DESCRIPTION
**Pull request by chunkhang**
To address #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,14 @@ tabline works and looks like.
...
 Vimrc file, using `let <optiona-name> = <option-value>`:
 
+* `g:buffet_always_show_tabline` - if set to `0`, the tabline will only be shown if
+  there is more than one buffer or tab open.
+
+  Default:
+  ```
+  let g:buffet_always_show_tabline = 1
+  ```
+
```
Comment (bagrat) [inline README.md]: There is always at least one tab open at any time, so you can limit this to just "more than one buffer open".

File: autoload/buffet.vim [hunk 2]
```diff
@@ -82,6 +82,11 @@ function! buffet#update()
     elseif s:last_current_buffer_id == -1 && len(s:buffer_ids) > 0
         let s:last_current_buffer_id = s:buffer_ids[0]
     endif
+
+    " Hide tabline if only one buffer and tab open
+    if !g:buffet_always_show_tabline && len(s:buffer_ids) == 1 && tabpagenr("$") == 1
+        set showtabline=0
+    endif
```
Comment (bagrat) [inline autoload/buffet.vim]: same here, you can drop the `tabpagenr("$") == 1` part.

File: README.md (hunk 1 above)
Comment (bagrat) [reply README.md]: I mean, it can never be 0 tabpages.

File: autoload/buffet.vim (hunk 2 above)
Comment (chunkhang) [reply autoload/buffet.vim]: we want to show the tabline if there's one buffer, but two or more tabs open

Comment (bagrat) [reply autoload/buffet.vim]: oh now I see, makes sense 👍

**Pull request merged**